src/torcontrol.cpp

```diff
@@ -608,7 +608,11 @@ void TorController::disconnected_cb(TorControlConnection& conn)
     service = CService();
     if (!reconnect)
         return;
-    LogPrintf("tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
+
+    if (LogAcceptCategory("tor")) {
+        LogPrintf("tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
+    }
```

Comment:
yes this would be equivalent:

    LogPrint("tor", "tor: Disconnected from Tor control port %s, trying to reconnect\n", target);